frontend/admin/src/js/lang/whitelist.yml

```diff
@@ -18,3 +18,5 @@
 - ev6HnY # 'Notes' CSV Header, Notes column
 - npC3bT # 'Notes' Title for admin editing a pool candidates notes
 - m0JFE/ # 'Code' Title for the application's profile snapshot
+- w/qZsH # 'Classification' Label displayed on the pool form classification field.
+- wHX/8C # 'Admin'  Title tag for Admin site
```

Comment:
```suggestion
- wHX/8C # 'Admin' Title tag for Admin site
```

Reply:
Good catch.  I've tidied it in [35a43b0](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4595/commits/35a43b04e1d6b0e9b99e451a2ffa05a4fed1b6fa).